src/backend.rs

```diff
@@ -13,23 +13,13 @@ use serde::Deserialize;
 use crate::config::Config;
 
```

Comment:
Is the derive Deserialize still required?

Reply:
You are right, it's not, will push shortly :)